components/cdc/src/endpoint.rs

```diff
@@ -886,13 +905,8 @@ impl Initializer {
             }
             debug!("cdc scan entries"; "len" => entries.len(), "region_id" => region_id);
             fail_point!("before_schedule_incremental_scan");
-            let scanned = Task::IncrementalScan {
-                region_id,
-                downstream_id,
-                entries,
-            };
-            if let Err(e) = self.sched.schedule(scanned) {
-                error!("cdc schedule cdc task failed"; "error" => ?e, "region_id" => region_id);
+            if let Err(e) = self.sink_scan_events(entries, done, require_barrier).await {
```

Comment:
Please confirm ticdc checks `RequestId` in release-4.0 too.

Reply:
Confirmed, release-4.0 can filter events by `RequestId`. https://github.com/pingcap/ticdc/blob/release-4.0/cdc/kv/client.go#L1243-L1257